README.md

```diff
@@ -1,61 +1,288 @@
+      <td align="left" width="25%">Demonstrates how the Graal compiler can achieve better performance for highly abstracted programs like those using Streams, Lambdas</td>
+      <td align="left" width="25%">Graal compiler, C2</td>
+      <td align="left" width="25%"><a href="https://www.graalvm.org/dev/examples/java-simple-stream-benchmark/">Simple Java Stream Benchmark</a></td>
+    </tr>
+    <tr>
+      <td align="left" width="25%"><a href="/streams/">streams</a><br><a href="https://github.com/graalvm/graalvm-demos/actions/workflows/streams.yml"><img alt="streams" src="https://github.com/graalvm/graalvm-demos/actions/workflows/streams.yml/badge.svg" /></a></td>
+      <td align="left" width="25%">Demontrates how GraalVM efficiently optimizes the Java Streams API application and how to apply PGO</td>
+      <td align="left" width="25%">Native Image, Native Build Tools Maven Plugin</td>
+      <td align="left" width="25%"><a href="https://www.graalvm.org/dev/reference-manual/native-image/guides/optimize-native-executable-with-pgo/">Optimize a Native Executable with Profile-Guided Optimizations</a></td>
+    </tr>
+    <tr>
+      <td align="left" width="25%"><a href="/fortune-demo/">fortune-demo</a><br><a href="https://github.com/graalvm/graalvm-demos/actions/workflows/fortune-demo.yml"><img alt="fortune-demo" src="https://github.com/graalvm/graalvm-demos/actions/workflows/fortune-demo.yml/badge.svg" /></a></td>
+      <td align="left" width="25%">A fortune teller Unix program. Run it in JIT, build a native executable, or build a mostly-static native executable, using Gradle or Maven build tools</td>
+      <td align="left" width="25%">Native Image, Native Build Tools</td>
+      <td align="left" width="25%"><a href="https://www.graalvm.org/dev/reference-manual/native-image/guides/use-graalvm-dashboard/">Use GraalVM Dashboard to Optimize the Size of a Native Executable</a></td>
+    </tr>
+    <tr>
+      <td align="left" width="25%"><a href="/multithreading-demo/">multithreading-demo</a><br><a href="https://github.com/graalvm/graalvm-demos/actions/workflows/streams.yml"><img alt="streams" src="https://github.com/graalvm/graalvm-demos/actions/workflows/streams.yml/badge.svg" /></a></td>
+      <td align="left" width="25%">Demonstrates how to optimize a Java application that does synchronous and asynchronous threads execution</td>
+      <td align="left" width="25%">Native Image, Native Build Tools Maven Plugin, GraalVM Dashboard</td>
+      <td align="left" width="25%"><a href="https://medium.com/graalvm/making-sense-of-native-image-contents-741a688dab4d">Making sense of Native Image contents</a></td>
```

Comment:
Should this be "making sense of the contents of a native executable"?

Reply:
No,  this is the blog post title, let's not modify it.